### ISSUE TITLE
The CIFAR-10-C dataset are not normalized

### ISSUE THREAD

**Issue by Newbeeyoung** [google-research/augmix#4]
I found that clean training and testing images are normalized, but why the test images in CIFAR-10-C are not normalized? Will it influence the performance?

```
test_data.data = np.load(base_path + corruption + '.npy')
test_data.targets = torch.LongTensor(np.load(base_path + 'labels.npy'))
test_loader = torch.utils.data.DataLoader(
        test_data,
        batch_size=args.eval_batch_size,
        shuffle=False,
        num_workers=args.num_workers,
        pin_memory=True)
```

**Comment by hendrycks** [comment]
There isn't a canonical baseline for CIFAR-10, so we just use a "uniform baseline" for normalizing, which is equivalent to averaging.
